src/org/labkey/workflow/WorkflowController.java

```diff
@@ -714,7 +714,7 @@ public void validateForm(WorkflowTaskForm form, Errors errors)
      * Delegates a task to a particular user.  The owner of the task remains unchanged.
      */
     @RequiresPermission(ReadPermission.class)
-    public class DelegateTaskAction extends ReadOnlyApiAction<WorkflowTaskForm>
+    public class DelegateTaskAction extends MutatingApiAction<WorkflowTaskForm>
     {
         @Override
```

Comment:
@labkey-susanh 
Looks like DelegateTaskAction is correctly extending MutatingApiAction, in which case, should it have something more than ReadPermission?